### PR TITLE
OPSEXP-146: add documentation on how to configure APS databases

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-146
Requires: #258

### CODE REVIEW

File: charts/alfresco-process-services/docs/databases-configuration.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Alfresco Process Service database configuration
+
+Alfresco Process Service uses a relational database to store data. An additional
+database is required in order to deploy the APS admin application.
+
+> It is not possible to use the same database for both for bot databases may be
```
Comment (pmacius) [inline charts/alfresco-process-services/docs/databases-configuration.md:6]: Typo `for both for bot`?

File: charts/alfresco-process-services/docs/databases-configuration.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-database-config
+data:
+  {{ template "alfresco-process-services.db.cm" .Values.apsdb }}
+```
+
+If you chose to not use the "alfresco-process-services.db.cm" elper template,
```
Comment (pmacius) [inline charts/alfresco-process-services/docs/databases-configuration.md:59]: elper -> helper

File: charts/alfresco-process-services/docs/databases-configuration.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+you need to make sure you also provide the `DATABASE_DRIVER` key in the
+configmap. The helper will try autodetect which class to use based on the URL
+provided.
+
+> If you want to use a custom driver class, you can provide it in the
+> `apsdb.driver` value.
+
+For the secret there is no helper and youneed to provide one seret which contains 2 data keys:
```
Comment (pmacius) [inline charts/alfresco-process-services/docs/databases-configuration.md:67]: youneed -> you need; seret -> secret

File: charts/alfresco-process-services/docs/elasticsearch-configuration.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Alfresco Process Services search index configuration
+
+Alfresco Process Services can use an Elasticsearch index. This document provides
+information on how to configure the search index in Alfresco Process Services.
+
+## Elasticsearch configuration
+
+It is quite common to tie Alfresco Process Services to an Elasticsearch index
+which lives outside of the Kubernetes cluster. For example Your index might be
```
Comment (gionn) [inline charts/alfresco-process-services/docs/elasticsearch-configuration.md:9]:
```suggestion
which lives outside of the Kubernetes cluster. For example your index might be
```

File: charts/alfresco-process-services/docs/elasticsearch-configuration.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+provided by a managed Elasticsearch service such as AWS OpenSearch.
+
+> APS for now only support ElasticSearch 7.x
+.
```
Comment (gionn) [inline charts/alfresco-process-services/docs/elasticsearch-configuration.md:13]:
```suggestion

```

**Review by gionn — APPROVED** [review]

**Pull request merged**